gyp_xwalk

```diff
@@ -351,6 +351,10 @@ if __name__ == '__main__':
     args.append('-Dnotifications=1')
     args.append('-Drelease_unwind_tables=0')
 
+  if gyp_vars_dict.get('OS') == 'win':
```

Comment:
Instead of adding more stuff to `gyp_xwalk`, please add those values to `build/common.gypi` with a proper comment about them. Adding stuff to `gyp_xwalk` is a historical mistake made when we started Crosswalk.